### PR TITLE
Feature/other meetings

### PR DESCRIPTION
**Pull request by JoeJasinski**
I think this is ready to merge. 

```
git push dev:feature/other-meetings

heroku app

heroku run --app chipy-dev /bin/bash
./manage.py migrate
./manage.py collectstatic
```

Example
https://chipy-dev.herokuapp.com/

### CODE REVIEW

**Comment by emperorcezar** [comment]
Could you fill out the description with which changes of the refresh are in here? Would help when I review.

**Comment by JoeJasinski** [comment]
@emperorcezar 

This ties back to https://github.com/chicagopython/chipy.org/issues/136.

There is still work to be done for that issue, but this will push out a ton of layout changes.

This includes:
 - a new Announcement model, and exposure via admin, which holds announcements that will display on the homepage banner
 - an announcement list view for displaying past announcements
 - a new meeting detail view, that allows direct linking to meetings
 - a new MeetingType model, allowing meetings to be associated with a type (i.e. Finance SIG, Holiday Party, etc)
 - a new meeting type foreign key attached to the Meeting model to support the above 
 - a new "group" model which allows SIGs to have their own basic landing page and associate meetings with them 
  - lots of changes to the homepage layout.  This is best visible by seeing: https://chipy-dev.herokuapp.com/
     - this includes bringing forward multiple upcoming meetings on the homepage
     - organizing the social links 
     -

**Comment by JoeJasinski** [comment]
@emperorcezar I made some slight updates to fix a bug and update some packages.  Should be ready for review

File: chipy_org/apps/meetings/templates/meetings/meeting.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{% extends "site_base.html" %}
+
+{% load i18n staticfiles %}
+
+{% block head_title %}{% trans "Topic" %}{% endblock %}
+
+{% block body_class %}home{% endblock %}
+
+{% block body %}
+<div itemscope itemtype="http://schema.org/Event">
```
Comment (emperorcezar) [inline chipy_org/apps/meetings/templates/meetings/meeting.html]: YAY! Thanks for using schemas

File: chipy_org/apps/meetings/views.py [hunk 2]
```diff
@@ -2,11 +2,7 @@
 
 from django.db.models import Sum
 from django.conf import settings
-from django.contrib.sites.models import Site
```
Comment (emperorcezar) [inline chipy_org/apps/meetings/views.py]: Thanks for the cleanup

**Comment by JoeJasinski** [comment]
Thanks @emperorcezar! Just saw this. I pushed out the changes.  @brianray The menu navigation has a new "host", "donate", and "sigs" menu items that are controlled by flapages. (so we can direct people toward more information about hosting, or donating or whatnot).  I can put in some text there if you like to get started, or I can defer to you on that.

**Pull request merged**